snapcraft/cli/__init__.py

```diff
@@ -100,7 +99,8 @@ def list_commands(self, ctx):
 
 
 @click.group(cls=SnapcraftGroup, invoke_without_command=True)
-@click.version_option(version=snapcraft.__version__)
+@click.version_option(message='snapcraft, version %(version)s',
```

Comment:
use the const defined in version

Reply:
this command changed to `( = MESSAGE_SNAPCRAFT_VERSION,version = snapcraft.__version__)